Patches/All-Tech Melee/Weapons_ATM_Industrial.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>All-Tech Melee</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- ========== Gladius ========== -->
+
+				<!-- Patch stats -->
+
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[contains(defName,
```

Comment:
This is a rather strange piece of xpath. Why do it this way versus the usual way, if I may ask? Is there something strange in the mod's xml that makes it necessary?

Reply:
It's a standard xpath, not sure what you mean by strange. Just wanted all of them to have the same cost in stuff.